templates/zerver/help/collapse-a-message.md

```diff
@@ -0,0 +1,8 @@
+## Collapse a message
+
+Zulip allows you to collapse messages for enhanced message viewing abilities.
+
+You can do this by simply hovering your mouse over the message, pressing the down chevron button near the upper right corner
+![down chevron](/static/images/help/down_chevron.png) and selecting Collapse.
```

Comment:
I would bold "Collapse" here so that it matches some of the other documentation pages.